resources/privacy-and-gdpr.md

```diff
@@ -6,53 +6,52 @@ Before implementing these controls, read Adobe's [GDPR documentation](https://ww
 
 When Adobe provides software and services to an enterprise, Adobe acts as a data processor for any personal data it processes and stores as part of providing these services. As a data processor, Adobe processes personal data in accordance with your company’s permission and instructions, as set out in your agreement with Adobe. As a data controller, you can use the Experience Platform SDKs to support GDPR retrieve and delete requests from your mobile apps.
 
-## Set and get privacy status
+## Set and get consent collect
 
-You can set a privacy status to ensure collection of data suits your user's preferences.
+You can set the collect consent status to ensure collection of data suits your user's preferences.
 
-| Extension | Opt In | Opt Out | Opt Unknown |
+| Extension | Collect (Yes) | Collect (No) | Collect (Pending) |
```

Comment:
Yes (y) | No(n) | Pending (p)

Reply:
I've changed the Yes -> y, No -> n, but I find not specifying which consent we are talking about could become confusing  and also redundant to specify yes/yes twice.

`Collect (y) | Collect (n)`

Reply:
But Collect (p) might not be too obvious or you plan to keep Collect (pending)?

Reply:
@stcheng-adobe thats a good point. I had Collect (p), but will change to Collect (pending)